string part
geometry_msgs/PoseStamped req_pose
---
sensor_msgs/JointState joint_pos